terraform/README.md

```diff
@@ -0,0 +1,28 @@
+# Terraform Cloud Setup
+
+We're packaging in terraform code to express all of the infrastructure needed in order to deploy this app to an environment. The best way to handle terraform on a team is to have a remote state management solution so that anyone who makes changes to the infrastructure codebase can have access to the state. I like terraform cloud. 
+
+## Setting up Terraform Cloud
+
+### AWS
+I'll be following the docs found [here](https://developer.hashicorp.com/terraform/tutorials/automation/github-actions).
```

Comment:
@usydusgduy123 you can comment directly on the file changes too. feel free to ask any questions or make any suggestions any time you're a reviewer on a pull request.